src/install.py

```diff
@@ -81,7 +81,7 @@ def install_webapp(*args):
         # update term.js
         cmd("cd webapp-lib/term; ./compile")
         wtype = 'debug' if args[0].debug else 'production'
```

Comment:
Thanks for your PR!  

It looks like there is still an args[0] in this line?

Reply:
I'm so careless...

Reply:
Pretty much everybody makes lots of mistakes when writing code no matter how long they try.   It's great to be writing code where one has the luxury of making mistakes (e.g., I hate writing code where i have to push it into production immediately and it must work or things get way worse -- that sucks).  Relax and enjoy.